.github/workflows/scripts/translation_check.py

```
@@ -1,239 +1,181 @@
-"""Validates i18n translation tags against locale JSON files."""
-
-from __future__ import annotations
-
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Ruff (0.14.10)

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)